docs/run_validator.md

```diff
@@ -1,20 +1,20 @@
 
 # Join Cyberd testnet as a Validator
 
-**Note** The current active testnet is `euler-5` (substitute <testnet_chain_id> with that value, do not forget to remove the `<` and the `>` symbols).
+**Note** The current active testnet is `euler-6` (substitute <testnet_chain_id> with that value, do not forget to remove the `<` and the `>` symbols).
 
 ## Prepare your server
```

Comment:
add note on dedicated server for stability @mrlp4

Reply:
add note.